### PR TITLE
feat(LH-67704) FTD Resource

### PR DESCRIPTION
**Pull request by weilueluo**
https://jira-eng-rtp3.cisco.com/jira/browse/LH-67704

### CODE REVIEW

File: provider/internal/util/togo.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package util
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+func TFStringListToGoStringList(l []types.String) []string {
+	res := make([]string, len(l))
```
Comment (bobertyy) [inline provider/internal/util/togo.go:6]: We could use the `Map` func here

**Review by bobertyy — APPROVED** [review]

File: provider/internal/device/ftd/operation.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	if err != nil {
+		return err
+	}
+
+	// map return struct to model
+	planData.ID = types.StringValue(res.Uid)
+	planData.Name = types.StringValue(res.Name)
+	planData.AccessPolicyName = types.StringValue(res.Metadata.AccessPolicyName)
+	planData.AccessPolicyUid = types.StringValue(res.Metadata.AccessPolicyUid)
+	//planData.Virtual = types.BoolValue(res.Metadata.PerformanceTier != nil)
```
[the remark this answers is not in the document]
Comment (weilueluo) [reply provider/internal/device/ftd/operation.go:71]: Thanks, fixed 👍

File: provider/internal/util/totf.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package util
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func GoStringSliceToTFStringList(stringSlice []string) []types.String {
+	l := make([]types.String, len(stringSlice))
```
[the remark this answers is not in the document]
Comment (weilueluo) [reply provider/internal/util/totf.go:8]: Yes good idea!

File: provider/internal/util/togo.go (hunk 1 above)
Comment (weilueluo) [reply provider/internal/util/togo.go:6]: Yes good idea!

**Pull request merged**